From: Michele Cane <michele.cane@gmail.com>
Subject: remove updater
Origin: vendor, Debian
Last-Updated: 2013-05-10

--- a/assets/prefs.js
+++ b/assets/prefs.js
@@ -79,7 +79,7 @@ pref("extensions.ui.lastCategory", "addo
 
 /** The below is imported from https://developer.mozilla.org/en/XULRunner/Application_Update **/
 // Whether or not app updates are enabled
-pref("app.update.enabled", true);
+pref("app.update.enabled", false);
 
 // This preference turns on app.update.mode and allows automatic download and
 // install to take place. We use a separate boolean toggle for this to make
--- a/modules/zotero/chrome/content/zotero/standalone/standalone.xul
+++ b/modules/zotero/chrome/content/zotero/standalone/standalone.xul
@@ -196,10 +196,12 @@
 									  accesskey="&helpReportErrors.accesskey;"
 									  label="&helpReportErrors.label;"
 									  command="cmd_zotero_reportErrors"/>
+							<!--
 							<menuitem id="checkForUpdates"
 									  accesskey="&helpCheckForUpdates.accesskey;"
 									  label="&helpCheckForUpdates.label;"
 									  oncommand="ZoteroStandalone.checkForUpdates();"/>
+							-->
 							<menuitem id="aboutName"
 									  accesskey="&aboutProduct.accesskey;"
 									  label="&aboutProduct.label;"
