From: Michele Cane <michele.cane@gmail.com>
Subject: remove browser on first start
Origin: vendor, Debian
Last-Updated: 2013-05-10

Index: zotero-standalone-build/modules/zotero/chrome/content/zotero/zoteroPane.js
===================================================================
--- zotero-standalone-build.orig/modules/zotero/chrome/content/zotero/zoteroPane.js	2013-11-16 20:46:04.274166108 +0100
+++ zotero-standalone-build/modules/zotero/chrome/content/zotero/zoteroPane.js	2013-11-16 20:46:04.270166100 +0100
@@ -224,7 +224,9 @@
 			if (Zotero.Schema.dbInitialized || !Zotero.Sync.Server.enabled) {
 				setTimeout(function () {
 					if(Zotero.isStandalone) {
+/*
 						ZoteroPane_Local.loadURI("https://www.zotero.org/start_standalone");
+*/
 					} else {
 						gBrowser.selectedTab = gBrowser.addTab("https://www.zotero.org/start");
 					}
