From: Michele Cane <michele.cane@gmail.com>
Subject: remove browser on first start
Origin: vendor, Debian
Last-Updated: 2013-05-10

--- a/zotero-client/chrome/content/zotero/zoteroPane.js
+++ b/zotero-client/chrome/content/zotero/zoteroPane.js
@@ -235,7 +235,9 @@
 		else if (Zotero.Prefs.get('firstRun2')) {
 			if (Zotero.Schema.dbInitialized || !Zotero.Sync.Server.enabled) {
 				setTimeout(function () {
+/*
 					ZoteroPane_Local.loadURI(ZOTERO_CONFIG.START_URL);
+*/
 				}, 400);
 			}
 			Zotero.Prefs.set('firstRun2', false);
