From: Michele Cane <michele.cane@gmail.com>
Subject: remove browser on first start
Origin: vendor, Debian
Last-Updated: 2013-05-10

--- a/modules/zotero/chrome/content/zotero/zoteroPane.js
+++ b/modules/zotero/chrome/content/zotero/zoteroPane.js
@@ -235,7 +235,9 @@ var ZoteroPane = new function()
 			if (Zotero.Schema.dbInitialized || !Zotero.Sync.Server.enabled) {
 				setTimeout(function () {
 					if(Zotero.isStandalone) {
+/*
 						ZoteroPane_Local.loadURI("https://www.zotero.org/start_standalone");
+*/
 					} else {
 						gBrowser.selectedTab = gBrowser.addTab("https://www.zotero.org/start");
 					}
