From: Michele Cane <michele.cane@gmail.com>
Subject: remove browser on first start
Origin: vendor, Debian
Last-Updated: 2013-05-10

Index: zotero-standalone-build/modules/zotero/chrome/content/zotero/zoteroPane.js
===================================================================
--- zotero-standalone-build.orig/modules/zotero/chrome/content/zotero/zoteroPane.js	2015-05-25 16:09:37.874385282 +0200
+++ zotero-standalone-build/modules/zotero/chrome/content/zotero/zoteroPane.js	2015-05-25 16:09:37.866385286 +0200
@@ -228,7 +228,9 @@
 			if (Zotero.Schema.dbInitialized || !Zotero.Sync.Server.enabled) {
 				setTimeout(function () {
 					if(Zotero.isStandalone) {
+/*
 						ZoteroPane_Local.loadURI("https://www.zotero.org/start_standalone");
+*/
 					} else {
 						gBrowser.selectedTab = gBrowser.addTab("https://www.zotero.org/start");
 					}
