From: Michele Cane <michele.cane@gmail.com>
Subject: do not ask installation from GUI
Origin: vendor, Debian
Last-Updated: 2013-04-23

--- a/zotero-standalone-build/modules/zotero-libreoffice-integration/resource/installer.jsm
+++ b/zotero-standalone-build/modules/zotero-libreoffice-integration/resource/installer.jsm
@@ -220,13 +220,19 @@
 				if(success) {
 					zpi.success();
 				} else {
-					openInstallationWizard();
+/*					openInstallationWizard();
+*/
+                                        zpi.success();
+
 				}
 			});
 		} else if(!zpi.failSilently || havePaths) {
 			// Otherwise, if there are installations and we are not failing silently, open the
 			// wizard
-			openInstallationWizard();
+/*			openInstallationWizard();
+*/
+                                        zpi.success();
+
 		}
 	}
 	
