From: Michele Cane <michele.cane@gmail.com>
Subject: remove button to install pdftotext and pdfinfo
Origin: vendor, Debian
Last-Updated: 2013-05-20

Index: zotero-standalone-build/modules/zotero/chrome/content/zotero/preferences/preferences_search.xul
===================================================================
--- a/zotero-client/chrome/content/zotero/preferences/preferences_search.xul	2013-05-16 22:09:51.000000000 +0200
+++ b/zotero-client/chrome/content/zotero/preferences/preferences_search.xul	2013-05-16 22:10:22.000000000 +0200
@@ -65,9 +65,9 @@
 			
 			<separator class="thin"/>
 			
-			<hbox>
+			<!--<hbox>
 				<button id="pdftools-update-button" flex="1" oncommand="Zotero_Preferences.Search.checkPDFToolsDownloadVersion()"/>
-			</hbox>
+			</hbox>-->
 			
 			<separator class="thin"/>
 			
